jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -2021,6 +2021,17 @@ static void loadLibrary(Class<?> caller, String name, boolean fullPath) {
 }
 
 /*[IF JAVA_SPEC_VERSION >= 15]*/
+/*[IF PLATFORM-mz31 | PLATFORM-mz64]*/
+static void loadZOSLibrary(Class<?> caller, String name) {
+	ClassLoader loader = (caller == null) ? null : caller.getClassLoader();
+	NativeLibraries nls = (loader == null) ? BootLoader.getNativeLibraries() : loader.nativelibs;
+	NativeLibrary nl = nls.loadZOSLibrary(caller, name);
+	if (nl == null) {
+		/*[MSG "K0647", "Can't load {0}"]*/
+		throw new UnsatisfiedLinkError(com.ibm.oti.util.Msg.getString("K0647", file));//$NON-NLS-1$
```

Comment:
`file` does not exist here, I think you meant `name`.

Reply:
Thanks, updated.